src/sql/orderbook/order_rewards.sql

```diff
@@ -110,20 +105,21 @@ order_surplus AS (
     JOIN auction_prices ap-- contains price: protocol fee token
         ON opf.auction_id = ap.auction_id AND opf.protocol_fee_token = ap.token
 ),
-     winning_quotes as (SELECT concat('0x', encode(oq.solver, 'hex')) as quote_solver,
-                               oq.order_uid
-                        FROM trades t
-                               INNER JOIN orders o ON order_uid = uid
-                               JOIN order_quotes oq ON t.order_uid = oq.order_uid
-                        WHERE (o.class = 'market'
-                            OR (o.kind = 'sell' AND (oq.sell_amount - oq.gas_amount * oq.gas_price / oq.sell_token_price) * oq.buy_amount >= o.buy_amount *  oq.sell_amount)
-                            OR (o.kind='buy' AND o.sell_amount >= oq.sell_amount + oq.gas_amount * oq.gas_price / oq.sell_token_price))
-                          AND o.partially_fillable='f' -- the code above might fail for partially fillable orders
-                          AND block_number > {{start_block}}
-                          AND block_number <= {{end_block}}
-                          AND oq.solver != '\x0000000000000000000000000000000000000000')
+winning_quotes as (
+    SELECT concat('0x', encode(oq.solver, 'hex')) as quote_solver,
+            oq.order_uid
+    FROM trades t
+            INNER JOIN orders o ON order_uid = uid
+            JOIN order_quotes oq ON t.order_uid = oq.order_uid
+    WHERE (o.class = 'market'
+        OR (o.kind = 'sell' AND (oq.sell_amount - oq.gas_amount * oq.gas_price / oq.sell_token_price) * oq.buy_amount >= o.buy_amount *  oq.sell_amount)
+        OR (o.kind='buy' AND o.sell_amount >= oq.sell_amount + oq.gas_amount * oq.gas_price / oq.sell_token_price))
+        AND o.partially_fillable='f' -- the code above might fail for partially fillable orders
+        AND t.block_number > {{start_block}}
+        AND t.block_number <= {{end_block}}
+        AND oq.solver != '\x0000000000000000000000000000000000000000')
 -- Most efficient column order for sorting would be having tx_hash or order_uid first
-select block_number,
+select trade_hashes.block_number as block_number,
```

Comment:
Does selecting the `block_number` differently change anything?

Reply:
Thought this was leftover redundant change from when i was trying to fix the test, I undid it but then needed to reapply it as indeed there is a complaint about the block_number being ambiguous. it should be the trade_hashes block number though as this is the main table we are left-joining with the rest.